packages/server/src/model/decoder.ts

```diff
@@ -761,3 +761,9 @@ export const NotificationDecoder = t.exact(
     eventId: t.number,
   })
 )
+
+export const SubscriptionsDecoder = t.exact(
+  t.type({
+    subscriptions: t.array(t.type({ objectId: Uuid, sendEmail: t.boolean })),
+  })
+)
```

Comment:
```suggestion
export const SubscriptionsDecoder = t.strict({
    subscriptions: t.array(t.type({ objectId: Uuid, sendEmail: t.boolean })),
  })
```

see  definition of `t.strict` in https://github.com/gcanti/io-ts/blob/master/index.md#implemented-types--combinators